src/bin/uhyve.rs

```diff
@@ -468,15 +477,34 @@ fn run_uhyve() -> i32 {
 
 	let mut app = Args::command();
 	let mut args = Args::parse();
+
 	// Tries to read arguments from a configuration file. If it doesn't exist or if
 	// parsing is not possible, continue using args as-is.
-	//
-	// TODO: Attempt to read configuration file from default locations (cwd, .config, etc.)
-	if let Some(config_file) = args.get_config_file()
-		&& let Ok(contents) = std::fs::read_to_string(config_file)
-		&& let Ok(toml_args) = toml::from_str::<'_, Args>(&contents)
+	if let Some(config_file) = args.get_config_file() {
+		if let Ok(toml_args) = read_toml_contents(config_file) {
+			args.merge(toml_args);
+		} else {
+			panic!("Could not parse config file.")
+		}
+	} else if let Ok(cwd) = std::env::current_dir()
+		&& let cwd_config = [cwd, "uhyve.toml".into()].iter().collect::<PathBuf>()
+		&& let Ok(toml_args) = read_toml_contents(&cwd_config)
```

Comment:
I think it would be a good idea to also raise a panic if any of the other config files are present but couldn't be parsed.